Polyhedron/demo/Polyhedron/Plugins/PMP/Scene_mcf_item.h

```diff
@@ -0,0 +1,50 @@
+#ifndef SCENE_MCF_ITEM_H
+#define SCENE_MCF_ITEM_H
+
+#ifdef scene_mcf_item_EXPORTS
+#  define SCENE_MCF_ITEM_EXPORT Q_DECL_EXPORT
+#else
+#  define SCENE_MCF_ITEM_EXPORT Q_DECL_IMPORT
+#endif
```

Comment:
That will not work. The code is compiled into two different libraries, named `scene_mcf_poly_item` and `scene_mcf_sm_item`. The macro `scene_mcf_item_EXPORTS` will never be defined.

You should add, before line 4:
```C++
#ifdef scene_mcf_sm_item_EXPORTS
#  define scene_mcf_item_EXPORTS
#endif
#ifdef scene_mcf_poly_item_EXPORTS
#  define scene_mcf_item_EXPORTS
#endif
```
or rewrite the line 4 as:
```C++
#if defined(scene_mcf_poly_item_EXPORTS) || defined(scene_mcf_poly_item_EXPORTS)
```